tests/ProtoTests/BasicLro/Gapic/BasicLroGapicClient.php

```diff
@@ -0,0 +1,214 @@
+ * calls that map to API methods. Sample code to get started:
+ *
+ * ```
+ * $basicLroServiceClient = new BasicLroClient();
+ * try {
+ *     $basicLroServiceClient->method1();
+ * } finally {
+ *     $basicLroServiceClient->close();
+ * }
+ * ```
+ *
+ * @experimental
+ */
+class BasicLroGapicClient
+{
+    use GapicClientTrait;
+
+    /** The name of the service. */
+    const SERVICE_NAME = 'testing.basiclro.BasicLro';
+
+    /** The default address of the service. */
+    const SERVICE_ADDRESS = 'lro.example.com';
+
+    /** The default port of the service. */
+    const DEFAULT_SERVICE_PORT = 443;
+
+    /** The name of the code generator, to be included in the agent header. */
+    const CODEGEN_NAME = 'gapic';
+
+    /** The default scopes required by the service. */
+    public static $serviceScopes = [
+        'scope1',
+        'scope2',
+    ];
+
+    private $operationsClient;
+
+    private static function getClientDefaults()
+    {
+        return [
+            'serviceName' => self::SERVICE_NAME,
+            'apiEndpoint' => self::SERVICE_ADDRESS . ':' . self::DEFAULT_SERVICE_PORT,
+            'clientConfig' => __DIR__ . '/../resources/basic_lro_client_config.json',
+            'descriptorsConfigPath' => __DIR__ . '/../resources/basic_lro_descriptor_config.php',
+            'gcpApiConfigPath' => __DIR__ . '/../resources/basic_lro_grpc_config.json',
+            'credentialsConfig' => [
+                'scopes' => self::$serviceScopes,
+            ],
+            'transportConfig' => [
+                'rest' => [
+                    'restClientConfigPath' => __DIR__ . '/../resources/basic_lro_rest_client_config.php',
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * Return an OperationsClient object with the same endpoint as $this.
+     *
+     * @return OperationsClient
+     *
+     * @experimental
+     */
+    public function getOperationsClient()
+    {
+        return $this->operationsClient;
+    }
+
+    /**
+     * Resume an existing long running operation that was previously started by a long
+     * running API method. If $methodName is not provided, or does not match a long
+     * running API method, then the operation can still be resumed, but the
+     * OperationResponse object will not deserialize the final response.
+     *
+     * @param string $operationName The name of the long running operation
+     * @param string $methodName    The name of the method used to start the operation
+     *
+     * @return OperationResponse
+     *
+     * @experimental
+     */
+    public function resumeOperation($operationName, $methodName = null)
+    {
+        $options = isset($this->descriptors[$methodName]['longRunning']) ? $this->descriptors[$methodName]['longRunning'] : [];
+        $operation = new OperationResponse($operationName, $this->getOperationsClient(), $options);
+        $operation->reload();
```

Comment:
Just to double check that reload doesn't return a new OperationResponse.

Reply:
Thanks for checking - this code is correct in that `reload` does not return a new OperationResponse (unlike C#).
This line is modelled from the monolithic generator here: https://github.com/googleapis/gapic-generator/blob/ae9338ab74e0fd91ee2814b348d59562693a76a6/src/main/resources/com/google/api/codegen/php/client_impl.snip#L303